src/transition.js

```diff
@@ -39,6 +39,29 @@ export function all(transitions) {
 
 
 /**
+ * Returns a transition that combines the string result of other string-based
+ * transitions such as transform and scale using the given opt_delimiter.
+ * @param {!Array<!TransitionDef<string>>} transitions
+ * @param {string=} opt_delimiter Default to a single whitespace.
+ * @return {!TransitionDef<string>}
+ */
+export function concat(transitions, opt_delimiter) {
+  opt_delimiter = opt_delimiter || ' ';
```

Comment:
This will preclude from using empty string as a delimiter? Is it ok? Alternatively, you can use `delimiter = ' '` in the arg list.